rmw_dds_common/include/rmw_dds_common/context.hpp

```diff
@@ -27,16 +27,27 @@
 namespace rmw_dds_common
 {
 
+/// Structure of data that a Context will need in any rmw implementation
+/// mapping one Participant to Multiple Nodes.
 struct Context
 {
+  /// Gid of the Participant that the context uses.
```

Comment:
@ahcorde @ivanpauno nit: 
```suggestion
  /// Global ID of the Participant that the Context uses.
```